client/packages/invoices/src/Returns/api/api.ts

```diff
@@ -3,14 +3,22 @@ import { Sdk } from './operations.generated';
 
 export const getReturnsQueries = (sdk: Sdk, storeId: string) => ({
   get: {
-    newSupplierReturnLines: async (lineIds: string[]) => {
+    newSupplierReturnLines: async (inboundShipmentLineIds: string[]) => {
```

Comment:
hmm i didn't quite look at this code yet, i think for outbound return it should be stockLines, I can explain in person, when we look at API again, no changes required just wanted to flag